trikot-datasources/datasources-flow-filesystem/src/commonMain/kotlin/com/mirego/trikot/datasources/flow/generic/FileSystemDataSource.kt

```diff
@@ -38,10 +44,14 @@ class FileSystemDataSource<R : ExpiringFlowDataSourceRequest, T>(
     }
 
     override suspend fun save(request: R, data: FlowDataSourceExpiringValue<T>?) {
+        if (shouldSkipRequest(request)) {
```

Comment:
Not sure I understand properly here. What is the difference between throwing and just not saving anything?

Reply:
Good point, only a bad copy paste. We should do nothing here and not throw.